cf-check/diagnose.c

```diff
@@ -39,6 +39,11 @@ size_t diagnose_files(
 #include <validate.h>
 #include <openssl/rand.h>
 
+/* NOTE: Must be in sync with LMDB_MAXSIZE in libpromises/dbm_lmdb.c. */
+#ifndef LMDB_MAXSIZE
+#define LMDB_MAXSIZE    104857600
```

Comment:
If this is not aligned, what happens?

Reply:
`cf-check diagnose/repair` will have bad information about DB usage and `cf-check repair` may rotate DBs that don't need it or miss some DBs that do need it, depending on the mismatch.

Reply:
Could consider adding a test for this. Can either grep both files and compare, or create a unit tests which compiles to binaries just to print the 2 constants and compare them.

Reply:
Yeah a test would be nice. But we have multiple instances of the same thing so I'll leave to a separate PR.